apps/linkedin-to-notion/src/background/index.ts

```diff
@@ -27,3 +29,19 @@ chrome.runtime.onMessage.addListener(async (msg) => {
     return await chrome.tabs.create({ url: 'https://www.linkedin.com/in/me/' });
   }
 });
+
+/**
+ * A function that display changes in the secure storage on the authData key
+ * Should be removed, only there for testing purposes
+ */
+const temp = async () => {
+  const storage = new SecureStorage({ area: 'local' });
+  await storage.setPassword('napoleon');
+  storage.watch({
+    authData: (c) => {
+      console.log(c.newValue);
+    },
+  });
+};
+
+temp();
```

Comment:
You can use `// TODO` to highlight the todo using this extension: https://marketplace.visualstudio.com/items?itemName=Gruntfuggly.todo-tree

Even if it's temp I'd call it `displayChangesInSecureStorage` and add more context to the console.log such as `console.log("Secure storage value", c.newValue);`

Reply:
Nice one, thanks! Useless atm (cf. longer comment below as we won't use secure storage for now) but I've installed it anyway!